### PR TITLE
Rename default optimizer setting

### PR DESCRIPTION
**Pull request by chriseth**

### CODE REVIEW

File: libsolidity/interface/OptimiserSettings.h [hunk 1]
```diff
@@ -45,7 +45,7 @@ struct OptimiserSettings
 		return s;
 	}
 	/// Standard optimisations.
-	static OptimiserSettings enabled()
+	static OptimiserSettings regular()
```
Comment (axic) [inline libsolidity/interface/OptimiserSettings.h]: The comment calls it standard, why not go with that as a name?

Comment (axic) [reply libsolidity/interface/OptimiserSettings.h]: Or default?

Comment (chriseth) [reply libsolidity/interface/OptimiserSettings.h]: The problem with default is that the default is not disable the optimizer ;)
Standard sounds good, too!

**Comment by chriseth** [comment]
Rebased and changed the name.

**Review by erak — APPROVED** [review]

**Pull request merged**